reporters_db/data/reporters.json

```diff
@@ -2094,14 +2095,17 @@
             },
             "examples": [
                 "Bayard's Notebook, 235",
-                "1 Bayard's Notebook, 235"
+                "1 Bayard's Notebook, 235",
+                "Bayard’s Notebook, 235"
             ],
             "mlz_jurisdiction": [
                 "us:de;"
             ],
             "name": "Bayard's Notebook",
             "notes": "Dates are rough, but wide, estimated from a series called Delaware Cases, which had these dates. See: http://www.worldcat.org/oclc/2882722",
-            "variations": {}
+            "variations": {
+                "Bayard’s Notebook": "Bayard's Notebook"
```

Comment:
is this how we decided to resolve this?  or are we going to handle utf8 somewhere else?

Reply:
@mlissner suggested when there are only a few variations with unicode character, insert it directly in "variations", and if there are many variations of the same reporter, handle it with regexes https://github.com/freelawproject/eyecite/issues/121#issuecomment-1228884288

Reply:
great.